packages/@react-spectrum/button/stories/Button.stories.tsx

```diff
@@ -102,6 +106,10 @@ export const UserSelect: ButtonStory = {
   }
 };
 
+export const PendingSpinner: ButtonStory = {
```

Comment:
For combination `style="outline"`  + `staticColor="black"`, the spinner color doesn't currently match the text/outline color.

<img width="289" alt="Screenshot 2023-08-28 at 12 42 05 PM" src="https://github.com/adobe/react-spectrum/assets/8961049/5993015e-46a1-4885-9b51-acdc491fb296">

Reply:
We could just not use the `overBackground` for this combination, or custom style it black via `UNSAFE_className`.

Reply:
Will custom style in a separate PR